elasticsearch/indices/participants.cmi.angio.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
I did not move the `participants` indices to templates because I feel they're too study-specific.